### ISSUE TITLE
Upgrading Cypress to take advantage of native Typescript support is causing pipeline issues (The GPU process has crashed 6 time(s))

### ISSUE THREAD

**Issue by Joelasaur** [cypress-io/cypress#8039]
### Current behavior:
We are currently using the [cypress-webpack-preprocessor](https://github.com/cypress-io/cypress-webpack-preprocessor) to handle our Typescript test files, and we are removing it to take advantage of the native Typescript support included in Cypress 4.10.0. This works fine on our local dev machines, but when ran in parallel on our Jenkins instance it is throwing the error 

`The GPU process has crashed 6 time(s)` 

followed by 

`The Test Runner unexpectedly exited via a exit event with signal SIGILL` 

I won't be posting the full debug log here for privacy, but it has been shared with the Cypress team through email. I will also add that **this error still happens when we disable parallelization.**

### Desired behavior:

Cypress runs on 4.10.0 as successfully without the cypress-webpack-preprocessor as it does now on 4.1.0 *with* the preprocessor. 

### Test code to reproduce

Any test written in typescript is fine, the issue is with running typescript tests on our Jenkins infrastructure. I can provide the groovy code and AWS instance information to the Cypress team as needed. 

### Versions

Cypress Version: 4.10.0
OS: Ubuntu 16.04
Docker Base Image (cypress tests run here): node:10.12

**Comment by jennifer-shehane** [comment]
Hey @Joelasaur, I want to make sure nothing has fallen through the cracks here in communication. The last email I see was from our side requesting more DEBUG logs on May 27. Let me know if there's some newer communication I'm missing - or please re-forward the email containing the logs to support@cypress.io. We should be able to help with this. Thanks!

**Comment by jennifer-shehane** [comment]
Our team says they do have the logs from a previous call made. So, we should have access. Thanks.

**Comment by jennifer-shehane** [comment]
@Joelasaur I'm beginning to think this is a duplicate of #7045. There is someone who has mentioned a workaround: https://github.com/cypress-io/cypress/issues/7045#issuecomment-662303410 Waiting for feedback from the team on this suggested solution.

**Comment by jennifer-shehane** [comment]
Great to hear! Closing as duplicate of #7045
